### PR TITLE
Features login tests

### PR DESCRIPTION
**Pull request by DVA09**

### CODE REVIEW

File: allure-report/data/attachments/143d9dc19230c095.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (Alex19777) [inline allure-report/data/attachments/143d9dc19230c095.txt]: Add autogenerated files to gitignore

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,328 @@
+package-lock.json
+package.json
+test.js
+Login_tests/logintest.js
+Login_tests/logintest.unsuccessful.js
+node_modules/.package-lock.json
+node_modules/.bin/rimraf
```
Comment (Alex19777) [inline .gitignore]: Use '*' to reduce amount of lines

File: test/pages/login.pages.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+class LoginPage {
```
Comment (Alex19777) [inline test/pages/login.pages.js]: Do you really need this class?

File: test/specs/login.spec.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+    // await $("#user-name").setValue("standard_user");
+
+    // await $("#password").setValue("secret_sauce");
+
+    // await $("#login-button").click();
+
+    // await $('.app_logo').getText();
+
+    // await expect($('.app_logo')).toHaveTextContaining('Swag Labs');
```
Comment (Alex19777) [inline test/specs/login.spec.js]: Remove or uncomment this lines

File: test/specs/login.spec_negative.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+    // await $("#user-name").setValue("standard_user");
+
+    // await $("#password").setValue("secret_sauce");
+
+    // await $("#login-button").click();
+
+    // await $('.app_logo').getText();
+
+    // await expect($('.app_logo')).toHaveTextContaining('Swag Labs');
```
Comment (Alex19777) [inline test/specs/login.spec_negative.js]: Same

File: allure-report/data/attachments/143d9dc19230c095.txt (hunk 1 above)
Comment (DVA09) [reply allure-report/data/attachments/143d9dc19230c095.txt]: Fixed it

File: .gitignore (hunk 2 above)
Comment (DVA09) [reply .gitignore]: I used '/' to ignore an entire directory with all its contents

File: test/specs/login.spec_negative.js (hunk 5 above)
Comment (DVA09) [reply test/specs/login.spec_negative.js]: Deleted

File: test/pages/LoginPage.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    get loginButtonCSS() {
+      return $("#login-button");
+    }
+  
+    get loginMessageCSS() {
+      return $(".app_logo");
+    }
+    
+
+
```
Comment (Alex19777) [inline test/pages/LoginPage.js:20]: Empty spaces

File: test/pages/LoginPage.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    async loginXpath(userName, password) {
+      await this.usernameTextboxXpath.setValue(userName);
+      await this.passwordTextboxXpath.setValue(password);
+      await this.loginButtonXpath.click();
+    }
+  
+    async checkTitle(title){
+      await expect(this.loginMessageCSS).toHaveTextContaining(title);
```
Comment (Alex19777) [inline test/pages/LoginPage.js]: Move assertion to test layer

File: test/specs/login_page.spec.js [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+describe("Sauce Tests", () => {
+
+  it("Login Test Standard", async () => {
+
+    browser.url(login.url);
+
+    await LoginPage.loginCSS(login.username[0], login.password);
+
+    await LoginPage.checkTitle(login.title);
```
Comment (Alex19777) [inline test/specs/login_page.spec.js]: Use soft assertion

File: test/pages/LoginPage.js (hunk 7 above)
Comment (DVA09) [reply test/pages/LoginPage.js]: Done it

File: test/specs/login_page.spec.js (hunk 8 above)
Comment (DVA09) [reply test/specs/login_page.spec.js]: Use it in the 'Login Test Empty', add an array with two parameters

**Review by Alex19777 — APPROVED** [review]

**Pull request merged**